src/main/java/de/tum/in/www1/artemis/web/rest/ParticipationResource.java

```diff
@@ -561,12 +550,12 @@ else if (quizExercise.isSubmissionAllowed()) {
     /**
      * GET /participations/:participationId/status: get build status of the user's participation for the "participationId" participation.
      *
-     * @param id the participation participationId
+     * @param participationId the participation participationId
      * @return the ResponseEntity with status 200 (OK) and with body the participation, or with status 404 (Not Found)
      */
-    @GetMapping(value = "/participations/{participationId}/status")
-    public ResponseEntity<?> getParticipationStatus(@PathVariable Long id) {
-        StudentParticipation participation = participationService.findOneStudentParticipation(id);
+    @GetMapping("/participations/{participationId}/status")
+    public ResponseEntity<?> getParticipationStatus(@PathVariable Long participationId) {
```

Comment:
The endpoint was not reachable as it could not map the value from the path, so I changed it

Reply:
please delete the whole end point

Reply:
I merged this changed from the develop